day3_avinash/mail_sender_function.js

```diff
@@ -0,0 +1,17 @@
+var nodemailer =require('nodemailer');
```

Comment:
file name should follow camel case mail_sender_function.js should be mailSenderFunction